config/plugins/visualizations/fits_image_viewer/package.json

```diff
@@ -11,6 +11,6 @@
         "parcel-bundler": "^1.4.1"
     },
     "scripts": {
-        "build": "cp -r node_modules/aladin-lite-galaxy static/dist && parcel build src/script.js -d static"
+        "build": "mkdir -p static/dist; cp -r node_modules/aladin-lite-galaxy static/dist/. && parcel build src/script.js -d static"
```

Comment:
A more robust approach other plugins take is to use https://github.com/sindresorhus/cpy-cli -- that's what I'd recommend here.  Nora is a bit different in that it just stages and doesn't build, but it's still a good example: https://github.com/galaxyproject/galaxy/blob/dev/config/plugins/visualizations/nora/package.json#L16

Reply:
thanks, I didn't find other plugins using it except nora like you mentionned but it seems good, it looks like it would prevent the difference in behavior between different instances we have been experiencing, will make the switch to cpy and update the pr